mlflow/telemetry/client.py

```diff
@@ -63,11 +55,17 @@ def _fetch():
                     self._is_stopped = True
                     _set_telemetry_client(None)
                 else:
-                    # If any telemetry records are generated before the config is loaded,
-                    # filter them by the condition defined in the config before exporting.
-                    with self._batch_lock:
-                        if self._pending_records:
-                            self._drop_disabled_records()
+                    # send the import record immediately after config is fetched
+                    # do not add if config is None
+                    self.add_record(
+                        Record(
+                            event_name=ImportMlflowEvent.name,
+                            timestamp_ns=time.time_ns(),
+                            status=Status.SUCCESS,
+                            duration_ms=0,
+                        ),
+                        should_send=True,
+                    )
```

Comment:
Instead of adding a new parameter, can we call `_send_batch` here?

```suggestion
                    )
                    self._send_batch()
```